.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
+<describe the changes you have made here>
```

Comment:
I like that. May be we should add `motivation, considered alternatives, ...`. Not sure if that is too much.